src/slack/vacuumlabs/constants.js

```diff
@@ -11,102 +11,162 @@ Do you want to order 3 Cool Coolers and 2 Cool Smartphones? Just tell me so:
 
 Happy shopping!
 
-PS: Feel free to contribute at https://github.com/vacuumlabs/eshop-api`
+PS: Feel free to contribute at https://github.com/vacuumlabs/eshop-api`;
+
+const SPINOFFS = [
```

Comment:
where does this list come from?  
are you sure its exhaustive at least for the moment?

Reply:
I got this list from this [Link](https://vacuumlabs.com/blog/lifevacuum/discover-7-vacuumlabs-spin-offs-and-the-people-behind-them)
Currently Richard provided me an updated list of spinoffs.